packages/constants/src/index.ts

```diff
@@ -179,6 +179,101 @@ export enum FORMATS {
     DEPTH_STENCIL = 34041,
 }
 
+// webdoc should handle the @property list for us now (test it, & fix it if it doesn't)
+/**
+ * WebGL internal formats, including compressed texture formats provided by extensions
+ *
```

Comment:
this is not showing up in the documentation

Reply:
I was a little lazy there, sorry :). I'll add the \@property tags since the webdoc PR won't get merged before this one.